### DISCUSSION TITLE
SD Stamina Vial value very high

### DISCUSSION THREAD

**Discussion by fidomullis** [pwillworth/dfkreport#17] [General]
I've noticed that Stamina Vial value in the ledger is ridiculously high. 20 stampots = US$51,760. That can't be right
![image](https://user-images.githubusercontent.com/113218662/189461242-552df3f6-4b4d-4215-b055-99aedd381933.png)

**Comment by pwillworth** [comment]
Hi, yeah, unfortunately the chaotic activity for the couple weeks after the harmony bridge hack caused a lot of weird prices to come out of the dex and get saved in the db.  I've been meaning to get in and clean some of that up for a while.  I'm going to be working on that today.  I started by cleaning up the bad Stamina vial prices saved during that period so those should be better already.  Keep in mind that if prices are not available on the date of the event, the current price will be used, so as this bad data is being removed, that will be the fallback.

**Reply by fidomullis** [reply]
Thanks for the prompt reply. Oh and thanks heaps for the tool! This is by far the best thing out there for Harmony taxes.
If I were to manually edit some of these outlier prices in the CSV file, where would I look for the in-game item pricing charts? Is there such a resource? Thanks again

**Reply by pwillworth** [reply]
That's great, glad you find it helpful - I would think the best resource for fixing outliers would be to check historic prices on [dexcreener](https://dexscreener.com/harmony/defikingdoms)

**Reply by fidomullis** [reply]
OK thanks, I'll try that
